src/akamai_cache_buster/bustCache.py

```diff
@@ -55,22 +55,44 @@ def main():
     
     #connect to akamai and validate
     initEdgeGridAuth()
+        
 
-    print(getYMLFromUrl("https://cloud.redhat.com/config/main.yml").get(appName).get("frontend").get("paths"))
+    paths = getYMLFromUrl("https://cloud.redhat.com/config/main.yml").get(appName).get("frontend").get("paths")
+    for path in paths:
```

Comment:
Eww also (I doubt this needs to be performant but...) iterating over the same collection twice is just a super nono.
for path in paths:
  ...
for path in paths:

Is almost always a mistake.

Reply:
I just want to make a note that the performance difference is O(2n) vs O(2n+1); this change by itself won't really make a difference, performance-wise, but it would make the code a bit more elegant.